stack.yaml

```diff
@@ -1,4 +1,5 @@
 resolver: lts-7.14
+
```

Comment:
https://github.com/tweag/guides/blob/master/style/Haskell.md#general-style, last paragraph of that section. Not that it's worth making a point about this one, but do be aware of that.